apps/journeys-admin/src/components/Editor/Slider/Settings/Drawer/CardTemplates/CardTemplateDrawer.stories.tsx

```diff
@@ -11,15 +11,17 @@ import { CardTemplateDrawer } from '.'
 const CardTemplateDrawerStory: Meta<typeof CardTemplateDrawer> = {
   ...journeysAdminConfig,
   component: CardTemplateDrawer,
-  title: 'Journeys-Admin/Editor/CardTemplateDrawer'
+  title: 'Journeys-Admin/Editor/Slider/Settings/Drawer/CardTemplateDrawer'
 }
 
 const Template: StoryObj<typeof CardTemplateDrawer> = {
   render: () => {
     return (
       <MockedProvider>
         <EditorProvider>
-          <Drawer />
+          <Drawer>
```

Comment:
Can add the title here?
```suggestion
          <Drawer title='Card Templates' >
```